modules/arcgis/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@deck.gl/arcgis",
+  "version": "0.0.1",
+  "main": "index.js",
+  "devDependencies": {
```

Comment:
That is a huge amount of devDependencies, are those really used? I don't see any webpack scripts...

Also a nit: We usually put `devDependencies` at the bottom, below dependencies.

Reply:
The devDependencies of submodules should go into the root package.json.

Reply:
I ended up removing all of them; none of them is needed here.